packages/code.rb

```diff
@@ -6,11 +6,13 @@ class Code < Package
   version '1.25'
   case ARCH
   when 'x86_64'
+    # code-stable-1534444642.tar.gz
     source_url 'https://go.microsoft.com/fwlink/?LinkID=620884'
-    source_sha256 '5856bbebf38aa05d584da4722869bbe507cf123f69f7ffab5f1532d73dbf3438'
+    source_sha256 '5ef2f82e1272b0008a446fc8f62f9693e2af7c83d5922883ecbfd8a9986fd40b'
   when 'i686'
+    # code-stable-1534444635.tar.gz 
     source_url 'https://go.microsoft.com/fwlink/?LinkID=620885'
```

Comment:
Can we use the GitHub page to get the sources? https://github.com/Microsoft/vscode/releases

That will stop us from downloading from a moving target and hitting checksum errors every time they update.

Reply:
@skiman6010: I think the issue here is the source_url is not really source but binary.  But you are right, we should be using a static url that specifies a version rather than a generic download link.  @JL2210, can you please look into this so we can solve the moving target issue?